en-us/give-up-optifine.md

```diff
@@ -39,8 +39,7 @@ Here is the list of OptiFine features that are supported in this modpack:
 | Custom Colors               | ✔️            | [Colormatic](https://www.curseforge.com/minecraft/mc-mods/colormatic)                                                                                                                         |
 | Custom GUIs                 | ✔️            | [OptiGUI](https://www.curseforge.com/minecraft/mc-mods/optigui)                                                                                                                               |
 | Custom Items                | ✔️            | [CIT Resewn](https://www.curseforge.com/minecraft/mc-mods/cit-resewn)                                                                                                                         |
-| Emissive Blocks             | ✔️            | [Continuity](https://www.curseforge.com/minecraft/mc-mods/continuity)                              |
-| Emissive Entities           | ✔️            | [Entity Texture Features](https://www.curseforge.com/minecraft/mc-mods/entity-texture-features-fabric)                                                                                        |
+| Emissive Textures           | ✔️            | [Continuity](https://www.curseforge.com/minecraft/mc-mods/continuity) (blocks), [Entity Texture Features](https://www.curseforge.com/minecraft/mc-mods/entity-texture-features-fabric) (entities) |
```

Comment:
I'm not sure if it is actually better to combine those, as they are provided by separate mods...

But if I did do that, I'd also want to combine some other things like animation/particle/other toggles.

Reply:
Well, emmisive textures are a single feature so it's better to have them combined. The toggles you mentioned are about different things each.